.travis.yml

```diff
@@ -43,13 +44,26 @@ jobs:
         - USE_MAVEN_REPO="true"
         - CMD="./.ci/travis.sh jacoco"
 
+    - jdk: openjdk11
+      env:
+        - DESC="codenarc analysis"
+        - CMD="./.ci/codenarc.sh"
+
     # Ensure that all modules are used in no exception configs
     - env:
         - DESC="ensure that all modules are used in no exception configs"
         - CMD1="export PULL_REQUEST=$TRAVIS_PULL_REQUEST"
         - CMD2="./.ci/validation.sh verify-no-exception-configs"
         - CMD="$CMD1 && $CMD2"
 
+before_script:
+  - |
+    if [[ $TRAVIS_CPU_ARCH == 'ppc64le' ]]; then
+        export JAVA_HOME=/usr/lib/jvm/adoptopenjdk-11-hotspot-ppc64el
```

Comment:
@romani Should this actually be in `./.ci/travis.sh install-adoptium-jdk` as it is the JRE and to keep it all together as needed for travis.

Reply:
Let's keep evidence of weird architecture in yml file, to remind us of this when adding new jobs here.